terraform/examples/io500/build_daos_io500_images.sh

```diff
@@ -0,0 +1,371 @@
+#!/bin/bash
```

Comment:
@markaolson I am not a big fan of this but ok.

Reply:
Thanks for letting me know.